ui/client/components/graph/node-modal/NodeDetailsContent.js

```diff
@@ -165,9 +175,7 @@ export class NodeDetailsContent extends React.Component {
             isMarked={this.isMarked}
             readOnly={!this.props.isEditMode}
             removeElement={this.removeElement}
-            toogleCloseOnEsc={this.props.toogleCloseOnEsc}
             showValidation={showValidation}
-            errors={fieldErrors}
```

Comment:
I see that errors are required in SubprocessInputDefinition but I understand it makes no sens in case of SubprocessInputDefinition - maybe lets make them optional or set default empty value